payload/builder.go

```diff
@@ -324,6 +326,58 @@ func (p *Payload) SoundVolume(volume float32) *Payload {
 	return p
 }
 
+// InterruptionLevelPassive sets the aps interruption-level to "passive" on the payload.
+// This is to indicate the importance and delivery timing of a notification.
+// See: https://developer.apple.com/documentation/usernotifications/unnotificationinterruptionlevel/
+//
+// {"aps":{"interruption-level":passive}}
+func (p *Payload) InterruptionLevelPassive() *Payload {
+	p.aps().InterruptionLevel = "passive"
+	return p
+}
+
+// InterruptionLevelActive sets the aps interruption-level to "active" on the payload.
+// This is to indicate the importance and delivery timing of a notification.
+// See: https://developer.apple.com/documentation/usernotifications/unnotificationinterruptionlevel/
+//
+// {"aps":{"interruption-level":active}}
+func (p *Payload) InterruptionLevelActive() *Payload {
+	p.aps().InterruptionLevel = "active"
+	return p
+}
+
+// InterruptionLevelTimeSensitive sets the aps interruption-level to "time-senstive" on the payload.
+// This is to indicate the importance and delivery timing of a notification.
+// See: https://developer.apple.com/documentation/usernotifications/unnotificationinterruptionlevel/
+//
+// {"aps":{"interruption-level":time-sensitive}}
+func (p *Payload) InterruptionLevelTimeSensitive() *Payload {
+	p.aps().InterruptionLevel = "time-sensitive"
```

Comment:
FWIW, Apple's documentation currently typos this as `time-senstive` (as does the comment above this comment).  

https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/generating_a_remote_notification#2943360

I have not yet tested which spelling is actually accepted on the device side, I'd be interested in knowing if you know that.

Reply:
I have just fixed the typo @froodian . Thank you for highlighting it.

Note that at the time of writing [Apple docs](https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/generating_a_remote_notification#2943360) have a typo, showing `time-senstive` as opposed to `time-sensitive`. 

Testing has shown that the correct spelling `time-sensitive` does indeed work.